lib/ui/widget/login_form.dart

```diff
@@ -100,12 +115,41 @@ class LoginFormState extends ConsumerState<LoginForm> {
               ),
             ),
             child: Text(AppLocalizations.of(context)!.login),
-            onPressed: () => {
-              // TODO: log in
-            },
+            onPressed: () => {_logIn()},
           ),
         ),
       ],
     );
   }
+
+  String? _emailValidator(String? value) {
+    if (value == null || !EmailValidator.validate(value)) {
```

Comment:
What do you think about moving this logic to the `LoginViewModel`? 🙏

Reply:
Resolved in https://github.com/nimblehq/ic-flutter-avishek/pull/36/commits/87733e49c74585d40d26767b5e685a67875506a2